test/unistd/links.c

```diff
@@ -41,11 +41,11 @@ void changedir(const char *dir) {
 }
 
 void setup() {
-  makedir("working");
+  makedir("/working");
 #if defined(__EMSCRIPTEN__) && defined(NODEFS)
-  EM_ASM(FS.mount(NODEFS, { root: '.' }, 'working'));
+  EM_ASM(FS.mount(NODEFS, { root: '.' }, '/working'));
 #endif
-  changedir("working");
+  changedir("/working");
```

Comment:
Why are any test changes needed at all to this file?

Reply:
The `@also_with_nodefs` mounts `/nodefs` and sets this as CWD on startup.

Without converting this test to use abs paths here that would break this test.

However I think using the absolute path (`/working`) is actually more correct as the check below assume that this is in the root directory.   See line 69

Reply:
Ideally we would have mount nodefs as `/` but I guess that maybe doesn't work for some reason?